_includes/how-to-guides/deployment/building-a-manifest-file.md

```diff
@@ -140,8 +140,12 @@ production:
 
 ### GlusterFS
 
+<div class="notice notice-warning"><p>
+The version of GlusterFS currently offered via the Cloud 66 Add-in is not supported by Ubuntu 18.04. To use this add-in please ensure that all of your application components are installed on servers running <strong>Ubuntu 16.04</strong>. 
+</p></div>
+
 - **version**: Specify the version of GlusterFS you want to install.
-- **operating_system** (_Optional_): `ubuntu1404`, `ubuntu1604` or `ubuntu1804`
+- **operating_system** (_Optional_): `ubuntu1404` or `ubuntu1604`
```

Comment:
Please remove ubuntu1404... I'm happy for users to reach out to support if they need it
Options should be ubuntu1604 or ubuntu1804 only